client/src/components/Input/InputInfo.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+import InputError from "./InputError";
+
+import { theme } from "constants/theme";
+
+const InputInfo = styled(InputError)`
+    color: ${props => props.error ? theme.colors.red : theme.colors.lightGray}
```

Comment:
Can you please use a darkey grey than this, it is barely visible
![image](https://user-images.githubusercontent.com/46847212/101206299-3a297f80-366f-11eb-9a52-6c571ca10919.png)
other than that, it looks good to me!